04-advanced-topics/network-analysis_rnaseq_01_wgcna.Rmd

```diff
@@ -526,21 +539,24 @@ Now we are ready for plotting.
 
 ```{r}
 ggplot(
-  module_52_df,
+  module_19_df,
   aes(
-    x = refinebio_treatment,
-    y = ME52,
-    color = refinebio_treatment
+    x = time_point,
+    y = ME19,
+    color = time_point
   )
 ) +
-  ggforce::geom_sina() +
+  ggforce::geom_sina(maxwidth = 0.3) +
+  # Putting on a boxplot, but making it see through and so we don't put outliers
+  # with boxplot
+  geom_boxplot(width = 0.2, alpha = 0, outlier.shape = NA) +
```

Comment:
If we change the order here, we don't need to use alpha
```suggestion
  # a boxplot with outlier points hidden (they will be in the sina plot)
  geom_boxplot(width = 0.2, outlier.shape = NA) +
  # A sina plot to show all of the individual data points
  ggforce::geom_sina(maxwidth = 0.3) +
```

Reply:
Sure. Why not.

Reply:
The comment changes too!